doc(v1.9.0): update notes for sle micro in longhorn command line interface installation (#1077)
longhorn/website: content/docs/1.9.0/v2-data-engine/quick-start.md

INFO[2024-01-01T00:00:10Z] Completed preflight installer. Use 'longhornctl check preflight' to check the result.
```

> **Note**:

> > Some immutable Linux distributions, such as SUSE Linux Enterprise Micro (SLE Micro), require you to reboot worker nodes after running the `install` sub-command. After the reboot, you will have to run the `install` sub-command again to complete the operation.

Reviewer: Do we know why need to execute `install` command again? I thought rebooting should be able to take effect after running `install` command? Or the second command is more about checking if the required dependencies are installed successfully?

Reviewer: No need. The PR here should be good enough.

> > Some immutable Linux distributions, such as SUSE Linux Enterprise Micro (SLE Micro), require you to reboot worker nodes after running the `install` sub-command. After the reboot, you will have to run the `install` sub-command again to complete the operation.

Reviewer 2: ```suggestion
> Some immutable Linux distributions, such as SUSE Linux Enterprise Micro (SLE Micro), require you to reboot worker nodes after running the `install` sub-command. After the reboot, you must run the `install` sub-command again to complete the operation.
```

Author: Addressed, thanks.

>

> > The documentation of the Linux distribution you are using should outline such requirements. For example, the [SLE Micro documentation](https://documentation.suse.com/sle-micro/6.0/html/Micro-transactional-updates/index.html#reference-transactional-update-usage) explains how all changes made by the `transactional-update` command become active only after the node is rebooted.

Reviewer 2: ```suggestion
> The documentation for your Linux distribution should outline such requirements. For example,For example, the [SLE Micro documentation](https://documentation.suse.com/sle-micro/6.0/html/Micro-transactional-updates/index.html#reference-transactional-update-usage) explains that all changes made by the `transactional-update` command become active only after the node is rebooted.
```

Author: Is the `For example, ` an error right?

Reviewer 2: yes.

It should be:
```
> The documentation for your Linux distribution should outline such requirements. For example, the [SLE Micro documentation](https://documentation.suse.com/sle-micro/6.0/html/Micro-transactional-updates/index.html#reference-transactional-update-usage) explains that all changes made by the `transactional-update` command become active only after the node is rebooted.
```
